third_party/NordicSemiconductor/libraries/crypto/ecp_alt_cc310.c

```diff
@@ -65,62 +69,42 @@
  *     <http://eprint.iacr.org/2004/342.pdf>
  */
 
+ /*
+  * Changes to original file:
+  * - Hardware accelerated EC point multiplication.
+  * - Hardware accelerated EC key pair generation.
+  *
+  * Added lines [496 - 749]
+  * Changed code in [2620 - 2631]
+  * Changed code in [3005 - 3013]
+  *
+  * Original file (ecp.c) from d369956d42764dc8173a8bf8106ed0f1defedec6
+  */
+
+
 #if !defined(MBEDTLS_CONFIG_FILE)
 #include "mbedtls/config.h"
 #else
 #include MBEDTLS_CONFIG_FILE
 #endif
 
-#if defined(MBEDTLS_ECP_ALT)
+#if defined(MBEDTLS_ECP_C)
 
 #include "mbedtls/ecp.h"
 #include "mbedtls/threading.h"
+#include "mbedtls/platform_util.h"
 
 #include <string.h>
 
-#include "cc310_mbedtls.h"
-#include "crys_ecpki_kg.h"
-#include "crys_ecpki_domain.h"
-#include "crys_rnd.h"
-#include "ssi_bitops.h"
-
-extern CRYSError_t PkaEcWrstScalarMult(const CRYS_ECPKI_Domain_t *pDomain,
-                                       const uint32_t            *scalar,
-                                       uint32_t                   scalSizeInWords,
-                                       uint32_t                  *inPointX,
-                                       uint32_t                  *inPointY,
-                                       uint32_t                  *outPointX,
-                                       uint32_t                  *outPointY,
-                                       uint32_t                  *tmpBuff);
-
-extern CRYS_RND_State_t  * pRndState;
-
-#define CC_CALLOC_CHECK(ptr, size)                      \
-do                                                      \
-{                                                       \
-    if (( ptr = mbedtls_calloc( 1, size ) ) == NULL )   \
-    {                                                   \
-        ret = MBEDTLS_ERR_ECP_ALLOC_FAILED;             \
-        goto cleanup;                                   \
-    }                                                   \
-} while (0)
-
-#define CC_CHECK_FREE(ptr)                              \
-do                                                      \
-{                                                       \
-    if (ptr != NULL)                                    \
-    {                                                   \
-        mbedtls_free( ptr );                            \
-    }                                                   \
-} while (0)
+#if defined(MBEDTLS_ECP_ALT)
 
 #if defined(MBEDTLS_PLATFORM_C)
 #include "mbedtls/platform.h"
 #else
 #include <stdlib.h>
 #include <stdio.h>
 #define mbedtls_printf     printf
-#define mbedtls_calloc     calloc
+#define mbedtls_calloc    calloc
```

Comment:
Not aligned

Reply:
I wanted to make as little as possible changes to original ecp.c file. That is why I haven't fixed the coding standard.